src/serlo-editor/plugins/equations/editor/editor.tsx

```diff
@@ -50,6 +47,10 @@ export function EquationsEditor(props: EquationsProps) {
     ) ||
     focusedElement === state.firstExplanation.id
 
+  const staticDocument = useAppSelector((storeState) =>
+    selectStaticDocument(storeState, id)
+  )
```

Comment:
`selectStaticDocument` only uses the `documents` slice and is cached by the `id`, so this really shouldn't affect performance at all. 👍